english/movie-ripoffs/zoolander.md

```diff
@@ -0,0 +1,23 @@
+As you approach the shaggy haired hippy, it is immediately obvious that the fellow is "high as a kite",
```

Comment:
It looks like this page is not reachable from the rest of the story. Make you sure you add a link to an existing page that leads to this page, or readers will not be able to find your addition.